appcircle_netrc/1.0.2/component.yaml

```diff
@@ -4,7 +4,7 @@ displayName: Authenticate with netrc
 description: "Add credentials to the .netrc file"
 docsUrl: https://docs.appcircle.io/workflows/common-workflow-steps#authenticate-with-netrc
 requiredComponents: 
-precedingComponents: 
+precedingComponents: "appcircle_git_clone"
 followingComponents: "appcircle_git_clone,appcircle_export_build_artifacts" 
```

Comment:
> @boztopuz Since this review is very repetitive, I will not give the same feedback in the rest, but can you provide the checks for all of them?

I mentioned the above situation in the previous review, but since the `export_build_artifacts` for which I did not add a comment were not corrected, I am adding them again for each one.
=> There is no any dependencies with `appcircle_export_build_artifacts`